test/e2e/destroypod_test.go

```diff
@@ -218,19 +246,27 @@ func TestDestroyPodTimely(t *testing.T) {
 }
 
 func TestDestroyPodWithRequests(t *testing.T) {
-	cancel := logstream.Start(t)
-	defer cancel()
-
+	// Not running in parallel as this test deletes activator pods
 	clients := Setup(t)
 
+	for _, tc := range testToDestroy {
+		t.Run(tc.name, func(t *testing.T) {
+			cancel := logstream.Start(t)
+			defer cancel()
+			testDestroyPodWithRequests(t, clients, tc.rmFunc)
+		})
+	}
+}
+
+func testDestroyPodWithRequests(t *testing.T, clients *test.Clients, rmFunc func(*test.Clients) error) {
 	names := test.ResourceNames{
 		Service: test.ObjectNameForTest(t),
 		Image:   "autoscale",
 	}
 	defer test.TearDown(clients, names)
 	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
 
-	objects, err := v1a1test.CreateRunLatestServiceReady(t, clients, &names, v1a1opts.WithRevisionTimeoutSeconds(int64(revisionTimeout.Seconds())))
+	objects, err := v1a1test.CreateRunLatestServiceReady(t, clients, &names, v1a1opts.WithRevisionTimeoutSeconds(int64(revisionTimeout.Seconds())), v1a1opts.WithConfigLabels(map[string]string{"knative.dev": "delete-to-test"}))
```

Comment:
As to my comment you want here tbc=-1 or you can't guarantee activator in the path.